src/py42/exceptions.py

```diff
@@ -194,12 +196,11 @@ def raise_py42_error(raised_error):
     if raised_error.response.status_code == 400:
         raise Py42BadRequestError(raised_error)
     elif raised_error.response.status_code == 401:
-        if raised_error.response.text:
-            if (
-                "TOTP_AUTH_CONFIGURATION_REQUIRED_FOR_USER"
-                or "TIME_BASED_ONE_TIME_PASSWORD_REQUIRED" in raised_error.response.text
-            ):
-                raise Py42MFARequiredError(raised_error)
+        if raised_error.response.text and re.search(
```

Comment:
i see the error!  Ha.  I feel like there are missing tests, we should add them so that this never happens again

Reply:
I think there are tests for this, just always (incorrectly) passed -- but if not, then yes, we should add some.

Reply:
Could also add tests to make sure that the error is _not_ raised under the right conditions.

Reply:
Oh I see, well then we would just a test to make sure that this _doesn't_ happen.

Reply:
Looking at it, I only see one test for `TIME_BASED_ONE_TIME_PASSWORD_REQUIRED` but not the other constant, and then there should be 401 test that does not have either, so 2 total missing tests I believe!

Reply:
Yeah, there is a test but it just always passed. I can add a negative test too.